tools/generator/src/DTO/Project.swift

```diff
@@ -11,6 +11,7 @@ struct Project: Equatable {
     let replacementLabels: [TargetID: BazelLabel]
     let targetHosts: [TargetID: [TargetID]]
     let envs: [TargetID: [String: String]]
+    let args: [TargetID: [String]]
```

Comment:
nit: can you have `args` above `env` everywhere they both exist?

Reply:
Will do